bin/amphtml-update.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -e
+
+# Go to the right location.
+if [[ '.' != $(dirname "$0") ]]; then
+	cd bin
+fi
+
+BIN_PATH="$(pwd)"
+PROJECT_PATH=$(dirname $PWD)
+VENDOR_PATH=$PROJECT_PATH/vendor
+
+if ! apt-get > /dev/null; then
```

Comment:
Better to check if the command exists via: `if ! command -v apt-get >/dev/null 2>&1; then`